### DISCUSSION TITLE
additional drag while gear extension

### DISCUSSION THREAD

**Discussion by Beginna78** [flybywiresim/aircraft#8559] [Ideas]
During the extension of the landing gear, it appears that there is noticeably more air drag due to large plates (only during the extension). Is that included in the current model?
[https://youtu.be/k1HCxoVKQ3k?si=1PR2_Kh3FUnmL2yn&t=2632](url)

**Comment by Maximilian-Reuter** [comment]
Hi, yes gear door drag is definitely simulated within the limits of the msfs flight model!
